sankaristoorit-ui/cypress/integration/tips_app.spec.js

```diff
@@ -23,6 +23,7 @@ describe('Tips app ', function () {
     cy.get('#password').type('passWord')
     cy.get('#login-button').click()
     cy.wait('@login')
+    cy.wait(200)
```

Comment:
Cy.wait(200) should not be needed if no other API calls are made between the intercept and wait